index.bs

```diff
@@ -2954,9 +2964,17 @@ when invoked, must run these steps:
 
 9. Let |targetRealm| be a user-agent defined [=Realm=].
 
-10. Let |clone| be <a abstract-op>StructuredClone</a>(|value|, |targetRealm|).
+10. Let |clone| be a [=clone=] of |value| in |targetRealm|.
     Rethrow any exceptions.
 
+    <details class=note>
+      <summary>Why create a copy of the value?</summary>
+      The value must be serialized when stored. Treating it as a copy
+      here allows other algorithms in this specification to treat it as
+      an ECMAScript value, but implementations may optimize this
+      if the difference in behavior is not observable.
+    </details>
```

Comment:
This difference can be observable, especially with very large ArrayBuffer objects. With those the initial copy might succeed, but then a later serialization could fail in a naive implementation. That seems somewhat bad.